.github/pull_request_template.md

```diff
@@ -1,10 +1,38 @@
-This PR implements the following **changes:**
+## summary
 
-* insert your summary of your request here, add more bullets as needed
+Basic description of work done.
```

Comment:
Consider linking it to an issue or trello card so we can track our work more easily.

```suggestion
Basic description of work done. Closes [#issue_no] or [trello_card_title].
```